api/src/main/java/org/pmiops/workbench/monitoring/StackdriverMonitoringService.java

```diff
@@ -0,0 +1,53 @@
+package org.pmiops.workbench.monitoring;
+
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.Measure;
+import io.opencensus.stats.Stats;
+import io.opencensus.stats.StatsRecorder;
+import io.opencensus.stats.View;
+import io.opencensus.stats.ViewManager;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class StackdriverMonitoringService implements MonitoringService {
+  private static final StatsRecorder STATS_RECORDER = Stats.getStatsRecorder();
+  private static final Logger log = Logger.getLogger(StackdriverMonitoringService.class.getName());
+
+  StackdriverMonitoringService() {
+    try {
+      StackdriverStatsExporter.createAndRegister();
+    } catch (IOException e) {
+      log.log(Level.SEVERE, "Failure to initialize stackdriver stats exporter");
+    }
+  }
+
+  @Override
+  public void registerSignal(
```

Comment:
In the audit code, I made sure that each method was hermetic in terms of exceptions. Thus, I catch all `RuntimeException`s from StackDriver and just log errors. We don't ever want a stackdriver outage to take down the system (though that scenario isn't common).